Buildings/Resources/src/ThermalZones/Detailed/Validation/BESTEST/simulateAndPlot.py

```diff
@@ -0,0 +1,988 @@
+#!/usr/bin/env python
+
+import matplotlib
+matplotlib.use('Agg')
+
+import matplotlib.pyplot as plt
+
+import io
+import json
+import os
+import sys
+import shutil
+
+# check if it just implements post-process
+POST_PROCESS_ONLY = False
+# check if delete the simulation result files
+CLEAN_MAT = True
+# check if delete the temporary working folders
+DelTemDir = True
+
+CWD = os.getcwd()
+FROM_GIT_HUB = True
+# Modelica Buildings Library working branch
+MBL_BRANCH = 'master'
+# BuildingSpy working branch. 
+# The working branch makes the communication points ('ncp') of jmodelica or optimica to be 8761,
+# so that the regression test will generate high resolution results.
+BP_BRANCH = 'issue335_high_ncp'
+# simulator, JModelica and optimica are supported
+TOOL = 'optimica'
+
+# standard data file
+ASHRAE_DATA = './ASHRAE140_data.dat'
+PACKAGES = ['Buildings.ThermalZones.Detailed.Validation.BESTEST.Cases6xx', \
```

Comment:
Can't you just use `Buildings.ThermalZones.Detailed.Validation.BESTEST`. Then BuildingsPy only need to be run once.

Reply:
updated